src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,30 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class DeleteHistories {
+
+    private final List<DeleteHistory> items;
+
+    public DeleteHistories(DeleteHistory param) {
+        this.items = Collections.singletonList(param);
```

Comment:
```suggestion
        this(Collections.singletonList(param));
```

https://github.com/next-step/jwp-qna/pull/486#discussion_r880350141 에 대한 답변 드립니다~
해당 코멘트는 위 코드와 같이 인스턴스 변수의 초기화를 한 생성자에서 하자! 라는 의미로 담긴 코멘트였는데요.

변경 가능성에 대한 내용으로 질문을 남겨주신 것 같아요 ㅎㅎ 
말씀해주신 부분도 정말 중요한 부분이고, 좋은 고민을 남겨주셨습니다 💯

Reply:
재연님 제가 아직 `인스턴스 변수의 초기화를 한 생성자에서 하자!`  라는 것에 대해 명확히 이해가 안되서 다시 질문드립니다ㅠ
인스턴스 변수의 초기화를 여러 생성자에서 하게되면 생길 수 있는 단점에 대해 궁금합니다
제가 생각하는 단점은 
`초기화 point가 여러군데라면 어디서 초기화되었는지 트래킹하기에도 end point 가 늘어나기 때문에 좋지 않다`
라고 생각을 했는데 어떻게 생각하시나요?? 
그리고 재연님이 생각하시는 단점도 궁금합니다!

Reply:
저도 동의합니다! 중복되는 코드를 만들었을 때 발생하는 단점과 동일하다고 생각해요 ㅎㅎ